examples/dial/main.go

```diff
@@ -21,7 +21,12 @@ func main() {
 	//
 
 	// Prepare the configuration of the DTLS connection
-	config := &dtls.Config{Certificate: certificate, PrivateKey: privateKey, InsecureSkipVerify: true}
+	config := &dtls.Config{
+		Certificate:          certificate,
+		PrivateKey:           privateKey,
+		InsecureSkipVerify:   true,
+		ExtendedMasterSecret: dtls.RequestExtendedMasterSecret,
```

Comment:
Do you think it is worth surfacing to the user the default value?

 I think it might even be ok to do `Require` and put a comment that you can/should lower to `Request` to support Clients/Servers that don't support it yet.

Reply:
Hmm yes I'm happy with making Require the default; but that means it can be somewhat of a breaking change

Reply:
Misunderstood - this is only for examples, fixed!